src/store_client.cc

```diff
@@ -659,18 +660,17 @@ storeClientCopyPending(store_client * sc, StoreEntry * e, void *data)
 {
 #if STORE_CLIENT_LIST_DEBUG
     assert(sc == storeClientListSearch(e->mem_obj, data));
+#else
+    (void)data;
 #endif
 #ifndef SILLY_CODE
-
     assert(sc);
 #endif
 
     assert(sc->entry == e);
 #if SILLY_CODE
-
     if (sc == NULL)
         return 0;
-
 #endif
```

Comment:
I think we should carefully remove SILLY_CODE instead of polishing its usage. I do not insist on this removal.

Reply:
I'll make a note and do a followup PR

Reply:
Your call, but I do not think a whole new PR is needed to remove two unused code lines and a pair of `#ifdef`s, especially since this PR already _modifies_ the lines that should be removed. If you decide that a separate PR is needed, then I would recommend removing all similarly unwanted `#ifdef`ed code in that dedicated PR.